README.md

```diff
@@ -748,6 +752,54 @@ The documentation on using Closure Stylesheets can be found
   `bazel run @io_bazel_rules_closure//closure/stylesheets -- --help`
 
 
+## closure\_proto\_js\_library
+
+```python
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_proto_js_library")
+closure_proto_js_library(name, srcs, add_require_for_enums, namespace_prefix,
+                         error_on_name_conflict, binary, import_style)
+```
+
+Defines a set of Protocol Buffer files.
+
+Documentation: [Protocol Buffers][protobuf] [JS][protobuf-js]
+[Generator Options](https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/js/js_generator.h).
+
+#### Implicit Output Targets
+
+- *name*.js: A generated protocol buffer JavaScript library.
+
+### Arguments
+
+- **name:** ([Name][name]; required) A unique name for this rule. Convention
+  states that such rules be named `foo_proto`.
+
+- **srcs:** (List of [labels][labels]; required) A list of `.proto` source
+  files that represent this library.
+
+- **add_require_for_enums:** (Boolean; optional; default is `False`) Add a
```

Comment:
Note: It seems like this flag is related to the PR you sent me a little bit ago, re: whether or not a `goog.require` is necessary for `@param` types. I would like for Closure Rules to be able to abstract this blemish in the future. We should probably ask the compiler / library guys if it would be possible for us to decide on *one* correct way to do it, and then enforce it in JsChecker. Maybe we should have a tracking issue for this.